### PR TITLE
Make sa token and certificates path configurable.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Follow up pr for https://github.com/tektoncd/results/pull/253

### Referenced issue
https://github.com/tektoncd/results/issues/256

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: test/e2e/01-install.sh [hunk 1]
```diff
@@ -17,6 +17,8 @@ set -e
 
 export KO_DOCKER_REPO=${KO_DOCKER_REPO:-"kind.local"}
 export KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-"tekton-results"}
+export SA_TOKEN_PATH=${SA_TOKEN_PATH:-"/tmp/tekton-results/tokens"}
+export SSL_CERT_PATH=${SSL_CERT_PATH:="/tmp/tekton-results/ssl"}
```
Comment (enarha) [inline test/e2e/01-install.sh:21]: That obviously works, but is there a reason you've used `:=` for the parameter expansion in this case, while `:-` is used in the other cases? The same goes in `e2e.sh`.

**Comment by adambkaplan** [comment]
/approve

@AndrienkoAleksandr looks like there is a merge conflict - please resolve and squash your commits.

**Comment by AndrienkoAleksandr** [comment]
> @AndrienkoAleksandr looks like there is a merge conflict - please resolve and squash your commits.

Done.

**Comment by adambkaplan** [comment]
/lgtm

Adding on behalf of @enarha

**Review by khrm — COMMENTED** [review]
/kind feature
We need this label for release as discussed in the last WG call.

**Pull request merged**